### DISCUSSION TITLE
Is there expiration at api key?

### DISCUSSION THREAD

**Discussion by jujumilk3** [Budibase/budibase#9893] [Help] [answered]
I'm building an api gateway for using budibase public api easily.
API keys are frequently used in my app, but there is no information about expiration anywhere, so I am asking about it.

Thank you.

**Comment by andz-bb** [comment, marked as answer]
hey @jujumilk3 - I don't believe the Public API keys are set to expire automatically. The only time it would change is if you manually regenerate it in the portal.
